doc/source/notebooks/intro_to_gpflow2.pct.py

```diff
@@ -510,17 +510,16 @@ def checkpointing_training_loop(
 # %%
 module_to_save = tf.Module()
 predict_fn = tf.function(
-    frozen_model.predict_f,
-    input_signature=[tf.TensorSpec(shape=[None, 1], dtype=tf.float64)],
-    autograph=False,
+    frozen_model.predict_f, input_signature=[tf.TensorSpec(shape=[None, 1], dtype=tf.float64)]
 )
 module_to_save.predict = predict_fn
 
 # %% [markdown]
 # Save original result for futher comparison
 
 # %%
-original_result = module_to_save.predict(samples_input)
+sample_input_tensor = tf.convert_to_tensor(samples_input, dtype=default_float())
```

Comment:
Could you add a comment explaining why you convert to tensor first ?

Reply:
I changed it during debugging and found out that using `np.array` with `tf.function` here is a bad example and I left it.